npm/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "The event-driven queue for any language.",
   "license": "SEE LICENSE IN LICENSE.md",
   "scripts": {
-    "prepublishOnly": "mkdir bin && touch bin/inngest",
+    "prepublishOnly": "mkdir bin && touch bin/inngest && cp ../README.md .",
```

Comment:
Can we add README to the `files` array in the package.json instead or is there something here that I'm not realizing 😄

Reply:
@djfarrelly It's just 'cause we're in the `npm` subdirectory here; you can't specify files outside of the root dir (i.e. it would be `../README.md`).

Either we add a package-specific `README.md` or copy across whatever the current root `README.md` is every time we publish.